src/server/report/app.js

```diff
@@ -1,35 +1,88 @@
 import './test-result.js';
-import { css, html, LitElement } from 'lit';
+import { css, html, LitElement, nothing } from 'lit';
 import data from './data.js';
 import page from 'page';
 
+const FILTER_STATUS = {
+	ALL: 'All',
+	PASSED: 'Passed',
+	FAILED: 'Failed'
+};
+
 class App extends LitElement {
 	static properties = {
+		_files: { state: true },
 		_filterFile: { state: true },
 		_filterTest: { state: true },
 		_mode: { state: true },
 		_overlay: { state: true }
 	};
 	static styles = [css`
+		.container {
+			display: grid;
+			grid-auto-flow: row;
+			grid-template-areas: "sidebar content";
+			grid-template-columns: 300px auto;
+		}
+		aside {
+			background-color: #fff;
+			border: 1px solid #e6e6e6;
+			box-shadow: 0 0 6px rgba(0,0,0,.07);
+			box-sizing: border-box;
+			grid-area: sidebar;
+			height: 100vh;
+			position: sticky;
+			top: 0;
+			z-index: 10;
+		}
+		aside > div {
+			padding: 10px;
+		}
+		main {
+			background-color: #fafafa;
+			grid-area: content;
+		}
+		main > div {
+			padding: 20px;
+		}
 		table {
+			background-color: #ffffff;
 			border-collapse: collapse;
 		}
 		td, th {
-			border: 1px solid #cdd5dc;
+			border: 1px solid #dfe6ef;
 			padding: 10px;
+			text-align: center;
 		}
 		thead th {
-			background-color: #f9fbff;
+			background-color: #f5f5f5;
 		}
 		tbody th {
 			font-weight: normal;
 			text-align: left;
 		}
+		td.passed {
+			background-color: #EFFFD9;
+		}
+		td.failed {
+			background-color: #FFEDE8;
+		}
```

Comment:
```suggestion
		td.passed {
			background-color: #efffd9;
		}
		td.failed {
			background-color: #ffede8;
		}
```
_Super_ minor lol, and I don't care if we go lowercase or uppercase, but just that they're the same

Reply:
For sure -- I was copying and pasting from the Chrome Dev Tools from the Daylight Site and wasn't paying super close attention.